corehq/apps/cloudcare/templates/cloudcare/partials/menu/badge.html

```diff
@@ -6,6 +6,6 @@
   and is different from x === undefined.
   */ %>
   <% if (typeof badgeText !== 'undefined' && badgeText) { %>
-  <span class="badge bg-secondary rounded-pill"><%- badgeText %></span>
+  <span id="module-badge" class="badge bg-dark rounded-pill"><%- badgeText %></span>
```

Comment:
Why not use a bootstrap font size class? `fs-6` or `fs-5` maybe.

Reply:
+1 to this ^